spec/src/main/asciidoc/ch03-entity-operations.adoc

```diff
@@ -1844,112 +1844,104 @@ it to be applied to that entity or mapped superclass and its subclasses.
-implementation dependent as to whether the pre-update validation event
-occurs. Portable applications should not rely on this behavior.
+In the same way as with entity lifecycle events (<<a2202>>),
+the number of times automatic validation is triggered
+for an entity within a single transaction,
+when multiple operations are performed on the same entity,
+depends on the Jakarta Persistence provider.
+
+For example, whether a pre-update validation event occurs
+after an entity is persisted and then modified,
+or after it is modified and then removed within the same transaction,
+is implementation-dependent.
+
+Portable applications must not rely on the occurrence or order
+of such automatic validation events.
 ====
 
 ===== Enabling Automatic Validation [[a2374]]
 
-The `validation-mode` element of the
-`persistence.xml` file determines whether the automatic lifecycle event
-validation is in effect. The values of the `validation-mode` element are
-`AUTO`, `CALLBACK`, `NONE`. The default validation mode is `AUTO`.
-
-If the application creates the entity manager
-factory using the `Persistence.createEntityManagerFactory` method, the
-validation mode can be specified using the
-`jakarta.persistence.validation.mode` map key, which will override the
-value specified (or defaulted) in the `persistence.xml` file. The map
-values for this key are _"auto"_, _"callback"_, _"none"_.
-
-If the auto validation mode is specified by
-the `validation-mode` element or the `jakarta.persistence.validation.mode`
-property, or if neither the `validation-mode` element nor the
-`jakarta.persistence.validation.mode` property is specified, and a Bean
-Validation provider is present in the environment, the persistence
-provider must perform the automatic validation of entities as described
-in <<a2380>>. If no Bean Validation provider is
-present in the environment, no lifecycle event validation takes place.
-
-If the callback validation mode is specified
-by the `validation-mode` element or the
-`jakarta.persistence.validation.mode` property, the persistence provider
-must perform the lifecycle event validation as described in <<a2380>>.
-It is an error if there is no Bean Validation
-provider present in the environment, and the provider must throw the
-`PersistenceException` if the `jakarta.persistence.validation.mode`
-property value _"callback"_ has been passed to the
-`Persistence.createEntityManagerFactory` method.
-
-If the none validation mode is specified by
-the `validation-mode` element or the `jakarta.persistence.validation.mode`
-property, the persistence provider must not perform lifecycle event
-validation.
+This specification defines the following validation modes:
+
+* `auto` -- if the Jakarta Validation _provider_ is obtainable,
+then the persistence provider must perform the automatic validation of entities
+as described in <<a2380>>.
+Otherwise, no lifecycle event validation takes place.
+This is the default validation mode.
```

Comment:
You're not defining the term "provider", so no italics.

```suggestion
* `auto` -- if the Jakarta Validation provider is obtainable, then the 
  persistence provider must perform the automatic validation of entities
  as described in <<a2380>>. Otherwise, no lifecycle event validation 
  takes place. This is the default validation mode.
```